.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] [PEP8 compliance](https://www.python.org/dev/peps/pep-0008/)
+- [ ] Docstrings for all functions
```

Comment:
We could, but yikes - perusing the actual standard is equivalent to reading a small book. Intimidating! Perhaps directing them to imitate existing docstrings would be more friendly? "Docstrings for functions, with style consistent across the package" or something similar?